benchmarks/common.py

```diff
@@ -698,7 +695,7 @@ def run_one_model(
                 return 0
 
             try:
-                with optimize_ctx:
+                with accuracy_ctx:
```

Comment:
Just to emphasize - Using a different ctx for accuracy testing. It is same as `optimize_ctx` unless overridden while handling cmdline args.

For aot_autograd, we choose no-decomps